wavesrv/pkg/cmdrunner/cmdrunner.go

```diff
@@ -1102,6 +1107,526 @@ func SidebarRemoveCommand(ctx context.Context, pk *scpacket.FeCommandPacketType)
 	return &sstore.ModelUpdate{Screens: []*sstore.ScreenType{screen}}, nil
 }
 
+func prettyPrintByteSize(size int64) string {
+	gbSize := float64(size) / float64(1073741824)
+	if gbSize > 1 {
+		return fmt.Sprintf("%.2f Gigabytes", gbSize)
+	}
+	mbSize := float64(size) / float64(1048576)
+	if mbSize > 1 {
+		return fmt.Sprintf("%.2f Megabytes", mbSize)
+	}
+	kbSize := float64(size) / float64(1024)
+	if kbSize > 1 {
+		return fmt.Sprintf("%.2f Kilobytes", kbSize)
+	}
+	return fmt.Sprintf("%v Bytes", size)
+}
+
+func writeCmdStatus(ctx context.Context, cmd *sstore.CmdType, startTime time.Time, exitSuccess bool, outputPos int64) {
```

Comment:
Here is where I update the status indicators
@esimkowitz @sawka 
does this look correctly integrated?

I'm seeing on the terminal that the green check mark, the x and the running indicator works, and I also see the command in the running commands tab

I see there's also a `StatusIndicatorLevel_Output` in memops.go
Is that meant to be used when there is output to the terminal? not sure if I'm using that one correctly

Reply:
The `StatusIndicatorLevel_Output` is supposed to be set when there is new data written to stdout or stderr since the last time the user visited a tab. The `StatusIndicatorLevel` values are reset when a tab becomes active and then increment as running commands make updates. Wherever you receive an update packet that is not "Done", that should probably trigger an update to display `StatusIndicatorLevel_Output`.

Also, you should call `IncrementNumRunningCmds` or `IncrementNumRunningCmds_Update` and pass it a `1` when the command starts executing and `-1` when the command is finished executing, whether success or failure. This will indicate that the Running Commands spinner for the tab/workspace indicator should show. This is separate from the spinner that shows for a specific command line.